### PR TITLE
Update base OS version, add GHCR

### PR DESCRIPTION
**Pull request by halostatue**

### CODE REVIEW

**Review by thefury — APPROVED** [review]

File: Maintenance.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# kineticcafe/sqitch-pgtap Maintenance
```
Comment (mikestok) [inline Maintenance.md:1]: Maybe `s/kineticcafe/KineticCafe/g`?

File: Maintenance.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+Note that this file _must_ be condensed as it is passed as a JSON object in
+GitHub Actions:
+
+```sh
+jq -c < package-versions.json | sponge package-versions.json
+```
+
+### Update `pg_prove`
+
+To `pg_prove`, update `pg_prove.version` in `package-versions.json`. That
```
Comment (mikestok) [inline Maintenance.md:20]: Perhaps

> To update `pg_prove`, update the value of `pg_prove.version` in `package-versions.json`.

File: Maintenance.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+downloaded from [PGXN][].
+
+The scripts and functions used by pgTAP vary by PostgreSQL version, so the
+updated versions must be cached as `pgtap.tar`, which is committed to this repo.
+This cache file can be updated with `just get-pgtap` (this requires
+[casey/just][]).
+
+### Update `sqitch`
+
+To `sqitch`, update `sqitch.version` in `package-versions.json`. That version of
```
Comment (mikestok) [inline Maintenance.md:37]: Looks like an "update" is missing (To update `sqitch`, …)

File: Maintenance.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+        && rm -rf /var/cache/apk/* /tmp/*
+    ```
+
+  - Add a `COPY` line to the `package-pgtap` section:
+
+    ```dockerfile
+    COPY --from=build-pgtap-psql-<VERSION> /opt/pgtap/<VERSION> /opt/pgtap/<VERSION>
+    ```
+
+When a new version PostgreSQL version is updated, the `pgtap.tar` cache must be
```
Comment (mikestok) [inline Maintenance.md:74]: Perhaps

> If the update uses a new version of PostgreSQL, run `just get-pgtap` to refresh the `pgtap.tar` cache.

File: package-versions.json [hunk 5]
```diff
@@ -0,0 +1 @@
+{"pg_prove":{"version":"3.36"},"pgtap":{"version":"1.2.1","hashref":"56c591fc3bf6b2dba49e15739625916d49e3fc27"},"sqitch":{"version":"1.3.1"}}
```
Comment (mikestok) [inline package-versions.json:1]: Does the format of this file matter? For human readability this seems better to me:

```
{
  "pg_prove": {
    "version": "3.36"
  },
  "pgtap": {
    "version": "1.2.1",
    "hashref": "56c591fc3bf6b2dba49e15739625916d49e3fc27"
  },
  "sqitch": {
    "version": "1.3.1"
  }
}
```

**Review by mikestok — APPROVED** [review]

File: Maintenance.md (hunk 1 above)
Comment (halostatue) [reply Maintenance.md:1]: The Docker image name is `kineticcafe/sqitch-pgtap`. I’m not sure if image names are case-sensitive or not, but…

File: package-versions.json (hunk 5 above)
Comment (halostatue) [reply package-versions.json:1]: Yes. I documented it under the Maintenance doc indicating that it _must_ be condensed. Github Actions breaks if it doesn’t.

File: Maintenance.md (hunk 4 above)
Comment (halostatue) [reply Maintenance.md:74]: Resolving it a different way.

**Review by ohnit — COMMENTED** [review]
from what i understand...approved!

**Pull request merged**